Blockzilla/AppDelegate.swift

```diff
@@ -269,6 +269,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
             TelemetryEventObject.app, value: nil, extras: ["orientation": orientation])
     }
     
+    func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
+        switch userActivity.activityType {
+        case "org.mozilla.ios.Klar.eraseAndOpen":
+            browserViewController.resetBrowser(hidePreviousSession: true)
+        default: break
```

Comment:
Perhaps by default we should throw some kind of error saying "not implemented" that way it's not a silent failure if we forget to handle some userActivity in the future. Definitely not necessary, though.

Reply:
I think that throwing an error is probably unnecessary here. We could display a toast if you think that's helpful?

Reply:
I'll defer to your judgement on this one! I think a toast message would be overkill; I was referring to an error that would be visible to devs but not users in some way.